### ISSUE TITLE
Link to user profile in markdown 

### ISSUE THREAD

**Issue by shreyas1599** [git-touch/git-touch#87]
Consider [comment_item](https://github.com/git-touch/git-touch/blob/master/lib/widgets/comment_item.dart#L186). 
I'm trying to add a link to the profile. For example, @shreyas1599 should link to my profile. I used a simple regex to do this: 

`body.replaceAllMapped(new RegExp(r"@[^\s]+"), (match) {
          return "[${match[0]}](https://github.com/${match[0]
})";`

A couple of questions:

- Should I place this in `markdown_view.dart` or leave it in `comment_item.dart` itself?
- Also in [mardown_view.dart](https://github.com/git-touch/git-touch/blob/master/lib/widgets/markdown_view.dart#L46-L68) I see you've used a `github` prefix. For now, I could just push the relative path. 
- But in `router.dart` you've added a [TODO: /github](https://github.com/git-touch/git-touch/blob/master/lib/router.dart#L65).
- So should I try and replace all the existing github routes to use the prefix `github` or just push the relative path for now? Something like this:
`return Provider.of<ThemeModel>(context).push(context, "/${m['login']}");`
- I could first open a PR to change all the existing routes to use `/github` and then implement the above.

@pd4d10 I'd like your opinion on this. Thanks.

**Comment by pd4d10** [comment]
If the purpose is to replace `@somebody` with a link, it seems not enough to just tweak the [onTapLink](https://github.com/git-touch/git-touch/blob/master/lib/widgets/markdown_view.dart#L31) method.

We should add a plugin to [flutter_markdown](https://github.com/flutter/flutter_markdown) to identify the `@somebody` syntax, then turn it into a link.

**Comment by pd4d10** [comment]
> But in router.dart you've added a TODO: /github.

Yeah, it is a legacy todo. We should open an issue for this: Add `/github` prefix to GitHub related screens, just like other platforms.

**Comment by pd4d10** [comment]
> body.replaceAllMapped(new RegExp(r"@[^\s]+"), (match) { return "[${match[0]}](https://github.com/${match[0] })";

Oh, I missed this replacement process. This might also work but it seems flutter_markdown plugin would be more common.

**Comment by shreyas1599** [comment]
Sorry, I don't exactly understand. I've never written one before. By plugin do you mean, I use `flutter_markdown` and create a package that identifies the `@` symbol and converts it into a link? Can you give an example?

**Comment by pd4d10** [comment]
`MarkdownBody` has a property named `extensionSet`: https://pub.dev/documentation/flutter_markdown/latest/flutter_markdown/MarkdownBody-class.html

For examples see: https://pub.dev/packages/markdown#syntax-extensions

**Comment by pd4d10** [comment]
There is also a legacy issue related to this: Rendering `<img>` tag correctly. `<img>` seems to be frequently used.
